docs/operator-manual/metrics.md

```diff
@@ -150,7 +150,7 @@ spec:
 
 ## Dashboards
 
-You can find an example Grafana dashboard [here](https://github.com/argoproj/argo-cd/blob/master/examples/dashboard.json) or check demo instance
-[dashboard](https://grafana.apps.argoproj.io).
+You can find an example Grafana dashboard [here](https://github.com/argoproj/argo-cd/blob/master/examples/dashboard.json).\
+Another option is to get the dashboard from [grafana.com](https://grafana.com/grafana/dashboards/14584).
```

Comment:
The dashboard - despite stating otherwise on grafana.com - is not officially endorsed by Argo CD, and the dashboard maintainer is not affiliated with the Argo project.

Thus, I think we should _not_ link to it from the official docs.

Reply:
Hello @jannfis, you are right, but I think from user perspective its easier to just copy dashboard ID from grafana.com, import and visualize.

Maybe we can setup a workflow to update and upload to grafana.com automatically by tracking `https://github.com/argoproj/argo-cd/blob/master/examples/dashboard.json`?

Reply:
@devudopw I'd be interested in that automation... maybe a weekly upload GitHub workflow?